test/e2e/harbor_test.go

```diff
@@ -20,6 +23,19 @@ func TestCPackagesHarborInstallSimpleFlow(t *testing.T) {
 	runHarborInstallSimpleFlow(test) // other args as necessary
 }
 
+func TestCPackagesHarborNutanixKubernetes123SimpleFlow(t *testing.T) {
```

Comment:
Thank you for adding this. can we add for all supported versions of k8s. 1.21 to 1.24

Reply:
At this time we only plan on targeting one specific k8s versions for packages e2e to reduce load.